lib/util.ts

```diff
@@ -33,21 +33,26 @@ export function roundTime(chain: ChainInfo, round: number) {
 
 export type HttpOptions = {
     userAgent?: string
+    headers?: Record<string, string>
 }
 
+// taking a separate `userAgent` param for backwards compatibility
 export const defaultHttpOptions: HttpOptions = {
-    userAgent: `drand-client-${LIB_VERSION}`
+    userAgent: `drand-client-${LIB_VERSION}`,
+    headers: {
+        'Access-Control-Allow-Origin': '*'
```

Comment:
I don't like this: decryption of a script or something that's rendered to the user is a great way to do a XSS, right?
So we might want to not have this kind of headers, even less so in the default client

Reply:
updated to remove default header

Reply:
I know this is removed now, but generally speaking `Access-Control-Allow-Origin` is a header that the server is supposed to send, not the client.

The drand APIs however should send it because their API is safe to use from any origin with no session stuff affecting the responses.